code/integration/main/driver_timer.cpp

```diff
@@ -130,3 +142,31 @@ void FactoryTerminatedOneOrder()                                   //factory exe
 
   }
 }
+//********************************************************************************************************
+//Function AddStringToArray
+//inserts the last incoming valid order to the order array;
+//from this order array the factory will take one (the oldest one) after the current order processing is over
+//NewString: last incoming accepted order
+void AddStringToArray(String NewString)
+{
+  int LastEmptyPos;
+  for (int i=4; i>=0; i--)
```

Comment:
#Defines für Size wären hier die bessere wahl. Wenn wir 10 Orders probieren wollen müssen wir an mehreren Stellen den Code ändern

#define MAX_ORDERS  5

String OrderArray[MAX_ORDERS]

for (..; i<MAX_ORDERS;...